UnityProject/Assets/Scripts/Shuttles/MatrixMove.cs

```diff
@@ -353,9 +353,9 @@ private void SyncInitialPosition(Vector3 oldPos, Vector3 initialPos)
 		networkedMatrix.MatrixSync.SyncInitialPosition(oldPos, initialPos);
 	}
 
-	public void SyncPivot(Vector3 oldPivot, Vector3 pivot)
+	private void SyncPivot(Vector3 oldPivot, Vector3 pivot)
 	{
-
+		networkedMatrix.MatrixSync.SyncPivot(oldPivot, pivot);
```

Comment:
and this one?

Reply:
forgot it from the matrix PR, and might have been causing scene issues